contact/contact.css

```diff
@@ -88,4 +85,70 @@ header .dark-mode{
     margin-top: 40px;
     color: #ff9500;
     text-decoration: none;
+}
+
+@media only screen and (min-width :1300px){
```

Comment:
![image](https://user-images.githubusercontent.com/76267220/128757180-46272075-2821-46f5-af9b-e1e78dda4e8b.png)
!!!!!!!!!!!
why it looks like that !! try to improve it

Reply:
Done